core/src/cats/make_bareos_tables.in

```diff
@@ -110,18 +110,14 @@ fi
 
 case ${db_type} in
    sqlite3)
-      sqlite3 $* ${working_dir}/${db_name}.db < ${temp_sql_schema}
-      chmod 640 ${working_dir}/${db_name}.db
-      retval=0
+      echo "The SQLite database backend is deprecated. Please use PostgreSQL instead."
+      echo "Creation of Bareos SQLite tables failed."
```

Comment:
is this a failure? We didn't even try.
I would just remove the second line

Reply:
Well, by just giving a message about the deprecation the user might expect tables were created. I would suggest changing failed to aborted.